erts/etc/common/erlexec.c

```diff
@@ -651,6 +646,16 @@ int main(int argc, char **argv)
                         } while ((i+1) < argc && argv[i+1][0] != '-' && argv[i+1][0] != '+');
 		    }
 #endif
+                    else if (strcmp(argv[i], "-configfd") == 0) {
+                        if (i+1 >= argc)
+ 			    usage("-configfd");
+                        if ( strcmp(argv[i+1], "0") != 0 ) {
+			    add_arg(argv[i]);
+                        } else {
+                            add_args("-noshell", "-noinput", NULL);
```

Comment:
Any checks required for someone setting it to 1?

Reply:
Thanks for the question! Standard input (file descriptor 0) is a special case because the Erlang runtime system automatically tries to buffer its contents, if the user has not specified `-noinput`. This buffering conflicts with reading configuration data from file descriptor 0. There is no such conflict for 1 ( standard output) and 2 (standard error) so no checks are necessary for these file descriptors. However, the Erlang system will not be able to print to standard output if 1 is used as a configuration file descriptor (a configuration file descriptor is closed after the configuration has been read) so it is probably not a good idea to use 1 and 2 as arguments to `-configfd` in most cases.

Reply:
Yeah sounds like something we'd want to put in docs, since I assume that using these values means we can't warn (either to display the fd0 warning, or to show warnings about fd1 or fd2 themselves)

Reply:
I agree. I will write something in the documentation to warn users about using fd0, fd1 and fd2. I don't understand what you mean with "either to display the fd0 warning" . What is the fd0 warning?

Reply:
Uh my bad. I misremembered a warning about fd0, but it was only in docs.

Reply:
I have added a note in the documentation for `-configfd` about fd1 and fd2:

See: https://github.com/erlang/otp/commit/ee2920a5daac95f80e2b522db62acb7a3fc36d39#diff-9b388e03a9038a603ba9d7c1d1cb04d38bbdd341f4ef7b4a8452306d8fe8fb79R251